cleaning_services/static/description/index.html

```diff
@@ -0,0 +1,17 @@
+<div class="w-100 px-3" data-oe-protected="false" contenteditable="true">
+    <h2><strong>Odoo for Cleaning Services</strong></h2>
+    <p>This industry package includes all the essentials for a cleaning services company, from opportunity management, to field service, and invoicing.</p>
+    <h3><strong>Basics</strong></h3>
+    <ul>
+        <li>The <strong><font class="text-o-color-1">CRM App</font></strong> manages customer inquiries and tracks leads with efficient follow-ups.</li>
+        <li>The <strong><font class="text-o-color-1">Sales App</font></strong> creates quotations and sales orders, converting leads into transactions.</li>
+        <li>The <strong><font class="text-o-color-1">Field Service Tasks</font></strong> dispatches on-site cleaning tasks promptly upon order confirmation.</li>
+        <li>The <strong><font class="text-o-color-1">Invoicing App</font></strong> automates bill generation and tracks payments for seamless financial management.</li>
+        <li>The <strong><font class="text-o-color-1">Recurring Visits</font></strong> schedules regular cleaning visits automatically for consistent service delivery in the field service app.</li>
+    </ul>
+    <h3><strong>Customisations</strong></h3>
+    <ul>
+        <li><strong>App settings</strong></li>
+        <li><strong>Template products</strong></li>
+    </ul>
```

Comment:
This part is weird. I think we can copy the traditional sentence when there is no customization

Reply:
<h3><strong>Included customizations</strong></h3>
    <p>This industry relies only on standard Odoo features and does not include any customization.</p>